test/common/router/config_impl_test.cc

```diff
@@ -136,6 +136,14 @@ Http::TestRequestHeaderMapImpl genHeaders(const std::string& host, const std::st
 
 envoy::config::route::v3::RouteConfiguration
 parseRouteConfigurationFromV2Yaml(const std::string& yaml) {
+  envoy::config::route::v3::RouteConfiguration route_config;
+  TestUtility::loadFromYaml(yaml, route_config, true);
+  TestUtility::validate(route_config);
+  return route_config;
```

Comment:
Is this intended to be just clearer or functionally different? I think both versions will boost, since `do_boosting` is true by default.

Reply:
Previously I've implemented v3 loading differently, but I agree it isn't needed.
I've updated the function name (`parseRouteConfigurationFromYaml`, removed V2/3), and made sure it loads V2 and V3 correctly.